src/sql/base/browser/ui/taskbar/actionbar.ts

```diff
@@ -81,7 +84,7 @@ export class ActionBar extends ActionRunner implements IActionRunner {
 			} else if (event.equals(KeyCode.Escape)) {
 				this.cancel();
 			} else if (event.equals(KeyCode.Enter) || event.equals(KeyCode.Space)) {
-				// Nothing, just staying out of the else branch
+				this._triggerKeyDown = true;
```

Comment:
There's a problem (not a regression) where pressing escape in the link callout will cause the cell to be unselected as well. Can we fix that at this level, or does that have to be fixed elsewhere?

Reply:
That's not a problem we can fix in the actionbar, but likely we could use the same type of fix as this one for that. We can do that separately though - do you have an issue made for that already?